src/issue/screens/issue.screen.js

```diff
@@ -230,7 +230,7 @@ class Issue extends Component {
       navigation,
     } = this.props;
 
-    const isLoadingData = isPendingComments || isPendingIssue;
+    const isLoadingData = isPendingComments || isPendingIssue || false;
```

Comment:
Great! This decides [crash](https://user-images.githubusercontent.com/4408379/28553564-1e14e4e6-70fd-11e7-881a-b9a61e31bd19.png) the application when you open the issue.